packages/general-types/src/constants/slot.ts

```diff
@@ -140,33 +170,42 @@ export const SlotTypes: ObjectKeys = {
       label: 'Age',
       values: ['20 days old', 'nineteen years old', '4 weeks old', '8 months old', '45 years-old', 'one month old'],
     },
-    // {
-    //   name: SlotType.COLOR,
-    //   label: 'Color',
-    //   values: ['white', 'blue', 'green', 'black', 'brown', 'yellow', 'red', 'maroon', 'cyan', 'orange'],
-    // },
-  ],
-  [Language.DE]: [
     {
-      name: SlotType.NAME,
-      label: 'Name',
+      name: SlotType.CURRENCY,
+      label: 'Currency',
       values: [
-        'Schmidt',
-        'Müller',
-        'Daniel',
-        'Michael',
-        'Lucas',
-        'Michelle',
-        'Laura',
```

Comment:
personName is actually not allowed in spanish or any other language except for English and Chinese. This is currently not working on prod.